contracts/evmscript/EVMScriptRunner.sol

```diff
@@ -14,7 +14,9 @@ import "../apps/AppStorage.sol";
 contract EVMScriptRunner is AppStorage, EVMScriptRegistryConstants {
     using ScriptHelpers for bytes;
 
-    function runScript(bytes _script, bytes _input, address[] _blacklist) protectState internal returns (bytes output) {
+    event ScriptResult(bytes script, bytes input, bytes returnData);
```

Comment:
Logging the executor address (and indexing it) could be useful in this log.

Reply:
Added in 4c5000e.